libs/blocks/carousel/carousel.css

```diff
@@ -0,0 +1,3 @@
+.carousel {
+  display: none !important;
```

Comment:
We need this `!important` ?

Reply:
Yea, it's fighting with the pink outline thing I think.

Reply:
ultimately this is temp code anyways since once carousel is created for customer-stories the concept of hiding it will be removed.